akka-http/src/main/scala/akka/http/scaladsl/server/ExceptionHandler.scala

```diff
@@ -37,16 +37,29 @@ object ExceptionHandler {
         if (!knownToBeSealed) ExceptionHandler(knownToBeSealed = true)(this orElse default(settings)) else this
     }
 
+  private def log(verbosity: ErrorLoggingVerbosity, simple: ⇒ Unit, full: ⇒ Unit) =
+    verbosity match {
+      case ParserSettings.ErrorLoggingVerbosity.Off    ⇒
+      case ParserSettings.ErrorLoggingVerbosity.Simple ⇒ simple
+      case ParserSettings.ErrorLoggingVerbosity.Full   ⇒ full
+    }
+
   def default(settings: RoutingSettings): ExceptionHandler =
     apply(knownToBeSealed = true) {
       case IllegalRequestException(info, status) ⇒ ctx ⇒ {
-        ctx.log.warning(
-          "Illegal request {}\n\t{}\n\tCompleting with '{}' response",
-          ctx.request, info.formatPretty, status)
+        log(
+          settings.errorLoggingVerbosity,
+          ctx.log.warning("Illegal request completing with '{}' response", status),
+          ctx.log.warning("Illegal request {}\n\t{}\n\tCompleting with '{}' response", ctx.request, info.formatPretty, status))
+
         ctx.complete((status, info.format(settings.verboseErrorMessages)))
       }
       case NonFatal(e) ⇒ ctx ⇒ {
-        ctx.log.error(e, "Error during processing of request {}", ctx.request)
+        log(
+          settings.errorLoggingVerbosity,
+          ctx.log.error("Error during processing one of requests occurred due to '{}'", e.getMessage),
```

Comment:
Again, Simple is meant to be one liner. Are you unhappy with e.getMessage? How do we replace it for one liner?